cms-contentful/README.md

```diff
@@ -0,0 +1,22 @@
+Sunrise Java Contentful CMS
+===========================
+
+Module for [Sunrise Java](https://github.com/sphereio/commercetools-sunrise-java) with Contentful service. 
+
+### How to use it
+
+To implement Contentful data into sunrise project, 
+one need to create ContentfulCmsService object using API token and space id:
+
+`ContentfulCmsService cmsService = ContentfulCmsService.of("spaceId", "token");`
+
+It will automatically fetch all entries.
```

Comment:
I've made this README.md to be actual with current implementation. 
I will correct it. It is fetching now string content. I think we wanted to cache it. 
For assets it is only their url.
One of requirments for implementing cms-contentful was to make small PR, 
so I divided my work into 
1. current PR - make call to contentful, which will return necessary content.
2. create models, for faster searching
3. add synchronization - fetch all cms content onnce, check for changes every ?? minutes
and update data
4. add configuration model, where credentials, how often to sync data would be defined
5. (eventually) update integration tests with new account